src/mongoose_rabbit_worker.erl

```diff
@@ -27,36 +27,34 @@
 
 -behaviour(gen_server).
 
--export([start_link/0, instrumentation/2]).
+%% API
+-export([instrumentation/2]).
 
--export([init/1, handle_call/3, handle_cast/2, handle_info/2,
-         terminate/2]).
+%% gen_server callbacks
+-export([init/1, handle_call/3, handle_cast/2, handle_info/2, terminate/2]).
 
 -ignore_xref([start_link/0]).
```

Comment:
This is not needed anymore, since the function is removed.

Reply:
Good catch, I had it removed, but this change got lost during rebase.